src/luascript.cpp

```diff
@@ -2725,6 +2724,7 @@ void LuaScriptInterface::registerFunctions()
 
 	registerMethod("Player", "isPzLocked", LuaScriptInterface::luaPlayerIsPzLocked);
 
+	registerMethod("Player", "hasClient", LuaScriptInterface::luaPlayerHasClient);
```

Comment:
Does this change belong in this PR? You register this method but are not using it anywhere